test/e2e/run_security_labels_test.go

```diff
@@ -47,6 +47,12 @@ var _ = Describe("Podman generate kube", func() {
 		test1 := podmanTest.Podman([]string{"create", "--label", "io.containers.capabilities=sys_admin", "--name", "test1", "alpine", "echo", "test1"})
 		test1.WaitWithDefaultTimeout()
 		Expect(test1).Should(Exit(0))
+		stderr := test1.ErrorToString()
+		if IsRemote() {
+			Expect(stderr).To(BeEmpty())
+		} else {
+			Expect(stderr).To(ContainSubstring("Capabilities requested by user or image are not allowed by default: \\\"CAP_SYS_ADMIN\\\""))
```

Comment:
It surprised me that this works, root & rootless, across all Fedora and Debian, ... but it does.